pkgs/tools/security/pinentry-bemenu/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, meson, ninja, pkg-config, libassuan
+, libgpg-error, popt, bemenu }:
+
+stdenv.mkDerivation rec {
+  pname = "pinentry-bemenu";
+  version = "0.6.0";
```

Comment:
There's already a v0.7: https://github.com/t-8ch/pinentry-bemenu/releases/tag/v0.7.0

Reply:
Thanks! I've updated the version.